lib/handle-pull-request-change.js

```diff
@@ -8,6 +8,16 @@ const DEFAULT_OPTS = {
   commitsOnly: false
 }
 
+const CONFIG_FILE = 'semantic.yml'
+
+async function getConfigOrDefault (context, defaultConfig) {
```

Comment:
Hey @gr2m I know you've worked on `probot-config`. Can you take a peek at this PR?